include/unifex/let_value.hpp

```diff
@@ -58,8 +58,13 @@ struct _successor_receiver<Operation, Values...>::type {
   void set_value(SuccessorValues&&... values) && noexcept {
     cleanup();
     UNIFEX_TRY {
-      unifex::set_value(
-          std::move(op_.receiver_), (SuccessorValues &&) values...);
+      // Taking by value here to force a copy on the offchance the value
+      // objects lives in the operation state (e.g., just), in which
+      // case the call to cleanup() would invalidate them.
+      [this](auto... copies) {
+        unifex::set_value(
+            std::move(op_.receiver_), (decltype(copies) &&) copies...);
+      } ((SuccessorValues&&) values...);
```

Comment:
The comment for this lambda indicates that it must preceed the call to cleanup(), yes?

Reply:
🤦‍♂️